app/controllers/webhooks/repo_updates_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Webhooks
+  class RepoUpdatesController < BaseController
+    def create
+      return unless pushed_to_master?
+
+      # TODO: enable this once we're out of the monorepo
+      # sync_track
+
+      # TODO: remove this this once we're out of the monorepo
+      sync_tracks
+
+      render json: {}, status: :ok
+    end
+
+    private
+    def sync_track
+      Git::SyncTrack.(track)
+    end
+
+    def sync_tracks
+      Track.all.each do |track|
+        Git::SyncTrack.(track)
+      rescue StandardError => e
+        Rails.logger.error "Error syncing Track #{track.slug}: #{e}"
+      end
+    end
+
+    def track
+      Track.find_by(slug: track_slug)
+    end
+
+    def track_slug
+      params[:repo][:name]
+    end
+
+    def pushed_to_master?
+      params[:ref] == MASTER_REF
+    end
+
+    MASTER_REF = "refs/heads/master".freeze
+    private_constant :MASTER_REF
```

Comment:
So none of this should get leaked into the controller itself. The controller is responsible for taking the request and handing it to to something to process, and then rendering the response. It shouldn't have any logic about the handling itself. 

Defining these methods inside the controller also leak them into every other controller action that might live in this at some time. As a general (pretty-hard) rule, controllers should only contain methods for actions, and methods for shared code between those actions (e.g. auth or retrieval of record across things).

So this should be moved either inline in the controller method, or should be moved into `app/commands/webhooks/process_repo_update`. The second would be my preference in this scenario.

Reply:
(The command should probably take the `params[:repo]` and `params[:ref]` as params